.buildconfig.yml

```diff
@@ -3,6 +3,18 @@
 componentsVersion: 92.0.0
 # Please add a treeherder group in taskcluster/ci/config.yml if you add a new project here.
 projects:
+  compose-awesomebar:
+    path: components/compose/awesomebar
+    description: 'An awesomebar component showing search results matching text entered into the toolbar.'
```

Comment:
nit: For those, should we point out that these are alternative components i.e, we have production ready ones but these are new ones using compose?